template/vars/config.sample.yaml

```diff
@@ -0,0 +1,59 @@
+---
+#
+# Bootstrap configuration - This file is gitignored
+#
+
+# Github username (e.g. onedr0p)
+bootstrap_github_username:
+# Github repository (e.g. flux-cluster-template)
+bootstrap_github_repository_name:
+# Age Public Key (e.g. age15uzrw396e67z9wdzsxzdk7ka0g2gr3l460e0slaea563zll3hdfqwqxdta)
+bootstrap_age_public_key:
+# Choose your timezone (e.g. America/New_York)
+bootstrap_timezone: Etc/UTC
+# Email you want to be associated with the ACME account
```

Comment:
```suggestion
# Email you want to be associated with the ACME account (used for TLS certificates via letsencrypt.org)
```

Or similar might be helpful for folks to figure out quicker what this is used for